### DISCUSSION TITLE
Questions about the new panel system

### DISCUSSION THREAD

**Discussion by failable** [zed-industries/zed#6580] [Feature Requests]
1. How to automatically zoom like the before setting `expand` when the terminal panel is shown using `workspace::ToggleBottomDock`?
2. How to automatically focus after calling `workspace::ToggleRightDock` and `workspace::ToggleBottomDock` without a click or a call to `terminal panel::toggle focus`?
3. `To change the location for either the project panel or the terminal panel, right-click on their respective panel icon and select the desired dock.` Nothing happens when I following the instruction, the terminal stays in bottom when I select `Dock right`. How to mimic `dock::AnchorDockRight` and `dock::AnchorDockBottom` then? Are there commands for moving the terminal now?
--------------------
Is it also possible to download an older version of Zed for now?

**Comment by JosephTLyons** [comment]
1. As of 0.88.6, the terminal panel should remember your zoomed state.  So you can open the bottom dock, toggle zoom, and then close the dock, and all subsequent dock opens should open it with zoom toggled.  That being said, maybe we should introduce settings for the default zoom state of each panel, when toggling.
2. I dont think this is possible ATM.
3. a) It's definitely a bug if the dock doesn't change positions when you select a different panel.  Can you check to make sure your `settings.json` file doesn't have any errors?  Trailing commas are a big culprit, as they are hard to see in Zed ATM. b) We don't have bindings for these, but I will bring it up with the team.

Yes, you can download older versions of Zed - you would definitely want to turn off the auto-update setting though.

**Comment by failable** [comment]
Hi, 

1. `set to zoom, and then close it`, I think you mean `toggle` it here. If one close/exit the terminal/exit Zed, it won't be zoomed next time.
2. Found a workaround, use `terminal_panel::ToggleFocus` instead of `workspace::ToggleBottomDock`.
3. I can't reproduce after updated to 0.88.6 at the moment. Bindings would be great :)

**Reply by failable** [reply]
Tracking issue: https://github.com/zed-industries/community/issues/1614

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
